selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindText.java

```diff
@@ -103,6 +103,7 @@ public void launchFindFormByKeyboard() {
 
   /** wait the 'Find Text' main form is closed */
   public void waitFindTextMainFormIsClosed() {
+    loader.waitOnClosed();
```

Comment:
do we need that as well ?

Reply:
yes, this is from PR https://github.com/eclipse/che/pull/6762
@SkorikSergey didn't ported that change to che6 and it was already in my branch so I decided to keep it.